src/components/ListaAlumnos.js

```diff
@@ -0,0 +1,74 @@
+
+import {useEffect, useState} from "react";
+import axios from 'axios'
+import * as React from 'react';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import ButtonAppBar from "./AppBar";
+import { prodUrl } from '../utils/constants';
+
+
+const ListaAlumnos = (props) =>{
+    //const url = "listAlumnosActivos";
```

Comment:
No tengas código comentado en tu código

https://martinfowler.com/bliki/Yagni.html
https://www.scaleyourapp.com/yagni-you-arent-gonna-need-it-or-is-it-😊/